engine/src/main/java/org/terasology/config/PlayerConfig.java

```diff
@@ -16,15 +16,18 @@
 
 package org.terasology.config;
 
-import java.util.List;
-
-import org.terasology.engine.subsystem.rpc.DiscordRPCSubSystem;
 import org.terasology.nui.Color;
 import org.terasology.rendering.nui.layers.mainMenu.settings.CieCamColors;
 import org.terasology.utilities.random.FastRandom;
 import org.terasology.utilities.random.Random;
+import org.terasology.utilities.subscribables.AbstractSubscribable;
+
+import java.util.List;
+
+public class PlayerConfig extends AbstractSubscribable {
```

Comment:
I'd like to take the chance to put some docstring here. Will take a closer look at this file to answer the following questions:
- what does it do? what kind of data does it hold?
- how does subscription work? how should that be used?
- any general notes?

@DarkWeird feel free to share some thoughts on this, I'll take care of making the Englishz fancy 🤓 🙃